libs/commons/UCommon.ml

```diff
@@ -37,6 +37,7 @@ let pr2 s =
   UStdlib.prerr_string "\n";
   flush UStdlib.stderr
 
+(* FIXME: Get rid of that, not thread-safe. *)
```

Comment:
can we create github issues instead of FIXME?

Reply:
Yes, once merged we will create them -- but I guess they will only be relevant when we merge? 
Then we can remove the FIXME one by one. Easier tracking.

Reply:
Or where you thinking to eliminate the FIXME before merge? We could do that too.